src/components/DirectLinks/DirectLinks.constants.js

```diff
@@ -0,0 +1,18 @@
+export const LINKS = {
+  MIJN_BELASTINGEN: {
+    displayName: 'Mijn belastingen',
+    url: 'https://belastingbalie.amsterdam.nl',
+  },
+  // MIJN_PARKEREN: {
```

Comment:
Why are these commented out? Maybe use this link: `https://www.amsterdam.nl/parkeren-verkeer/parkeervergunning/`

Reply:
I didn't have all the url's yet. I have them now and will add them.